### PR TITLE
openshift_version: skip rpm version==image version on Atomic

### PR DESCRIPTION
**Pull request by giuseppe**
since there is no repoquery installed on Atomic Host.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

**Comment by giuseppe** [comment]
aos-ci-test

File: roles/openshift_version/tasks/main.yml [hunk 1]
```diff
@@ -86,9 +87,10 @@
   include: set_version_rpm.yml
   when: not is_containerized | bool
 
+- name: Set openshift_version for containerized installation
+  include: set_version_containerized.yml
+
```
Comment (ingvagabund) [inline roles/openshift_version/tasks/main.yml]: when: is_containerized | bool

Comment (ingvagabund) [inline roles/openshift_version/tasks/main.yml]: Otherwise this will run during rpm-based deployment

**Comment by ingvagabund** [comment]
aos-ci-test

**Comment by giuseppe** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by giuseppe** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by ingvagabund** [comment]
[merge]

**Pull request merged**